easy-to-use-api/klips-chart-api/src/components/Chart/index.ts

```diff
@@ -59,6 +59,8 @@ import {
 } from '../../util/Chart';
 
 import WKTParser from 'jsts/org/locationtech/jts/io/WKTParser';
+// @ts-ignore
```

Comment:
Why do we need this here?

Reply:
I ran into an error importing GeoJSONWriter (line 63) since the declaration file for GeoJSONWriter is missing. This was the easiest fix.

Reply:
Okay thanks, I only asked, because the import of the `WKTParser` seems to work without `ts-ignore`

Reply:
Yes, for the WKTParser types were declared.